### PR TITLE
Converting to pure unit tests

### PR DESCRIPTION
**Pull request by seankane-msft**

### CODE REVIEW

File: sdk/tables/azure-data-tables/tests/test_table.py [hunk 1]
```diff
@@ -47,7 +48,10 @@
 
 # ------------------------------------------------------------------------------
 
-class StorageTableTest(TableTestCase):
+class StorageTableTest(AzureTestCase, TableTestCase):
+
+    def __init__(self, *args, **kwargs):
+        super(StorageTableTest, self).__init__(*args, **kwargs)
```
Comment (lmazuel) [inline sdk/tables/azure-data-tables/tests/test_table.py:54]: You don't need this initi since you do nothing with it

**Review by iscai-msft — APPROVED** [review]

**Pull request merged**